src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -623,6 +655,7 @@ internal enum LoadPhase
     {
         None,
         PreBuild,
-        PostBuild
+        PostBuild,
+        PostPostBuild,
```

Comment:
What does this phase do?

---
In reply to: [77580084](https://github.com/dotnet/docfx/pull/670#discussion_r77580084) [](ancestors = 77580084,77579915)